modules/log6x-code-ex.adoc

```diff
@@ -0,0 +1,242 @@
+= Logging 6.0 Code Examples
+
+Code examples used in the wider Logging 6.0 documentation are hosted here for single sourcing.
+
+////
+This file is not intended to be included in whole. Includes from this file should be using tagged regions only.
+References:
+* https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#including-by-tags
+* https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions/
+////
+
+// Content template within commented out block.
+////
+// tag::tagname[]
+[source,yaml]
+----
+Content
+More Content
+----
+// end::tagname[]
+////
+
+
+// tag::filters-unchanged[]
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: my-forwarder
+spec:
+  serviceAccount:
+    name: my-account
+  filters:
+    - name: my-multiline
+      type: detectMultilineException
+    - name: my-parse
+      type: parse
+    - name: my-labels
+      type: openshiftLabels
+      openshiftLabels:
+        foo: bar
+  pipelines:
+    - name: my-pipeline
+      inputRefs:
+        - application
+      outputRefs:
+        - my-output
+      filterRefs:
+        - my-multiline
+        - my-parse
+        - my-labels
+  outputs:
+    - name: my-output
+      type: http
+      http:
+        url: http://my-log-output:80
+----
+// end::filters-unchanged[]
+
+// tag::filters-changed[]
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: my-forwarder
+spec:
+  serviceAccount:
+    name: my-account
+  filters:
+    - name: drop-filter
+      type: drop
+      drop:
+        - test:
+            - field: '.level'
+              matches: 'debug'
+    - name: prune-filter
+      type: prune
+      prune:
+        in:
+          - '.kubernetes.labels.foobar'
+        notIn:
+          - '.message'
+    - name: audit-filter
+      type: kubeAPIAudit
+      kubeAPIAudit:
+        omitResponseCodes:
+          - 404
+          - 409
+  pipelines:
+    - name: my-pipeline
+      inputRefs:
+        - application
+        - audit
+      outputRefs:
+        - my-output
+      filterRefs:
+        - drop-filter
+        - prune-filter
+        - audit-filter
+  outputs:
+    - name: my-output
+      type: http
+      http:
+        url: http://my-log-output:80
+----
+// end::filters-changed[]
+
+// tag::inputs-app-audit-infra[]
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: my-forwarder
+spec:
+  serviceAccount:
+    name: my-account
+  inputs:
+    - name: app-logs
+      type: application
+      application:
+        includes:
+          - namespace: my-ns1
+            container: my-app1
+        excludes:
+          - namespace: my-ns2
+            container: my-app2
+    - name: audit-logs
+      type: audit
+      audit:
+        includes:
+          - user:
+              - my-user1
+    - name: infra-logs
+      type: infrastructure
```

Comment:
inputs.infrastructure doesn't support include, excludes . it only support filter by sources
https://github.com/openshift/cluster-logging-operator/blob/master/bundle/manifests/observability.openshift.io_clusterlogforwarders.yaml#L624